platform/macos/os_macos.mm

```diff
@@ -230,13 +234,23 @@
 
 	delete_main_loop();
 
-	if (joypad_apple) {
-		memdelete(joypad_apple);
+#ifdef SDL_ENABLED
+	if (joypad_sdl) {
+		memdelete(joypad_sdl);
 	}
+#endif
 }
 
 void OS_MacOS::initialize_joypads() {
-	joypad_apple = memnew(JoypadApple());
+#ifdef SDL_ENABLED
+	joypad_sdl = memnew(JoypadSDL());
+	if (joypad_sdl->initialize() == OK) {
+		return;
+	}
+	// SDL init failed, fallback to the native driver
```

Comment:
Where is the code that falls back to the native driver? Is there a reason SDL would fail to initialise?

Reply:
I suppose this was copied from the previous PR which didn't remove the Godot "joypad" driver: https://github.com/godotengine/godot/pull/87925/files#diff-a3dac44e3d02774569e24f3cac9d6e3842fe626380e022f9d0090a503058828dR150

So indeed this comment is wrong now.

Reply:
No worries – wanted to make sure it wasn't missed.